Update "Import browser data" - When your imported data is sent to Microsoft (#2934)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time. You can import your browser data to Microsoft Edge from another browser. The data includes your favorites, browsing history, cookies, autofill data, extensions, settings, and other browsing data.

With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.

Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 

* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.

> * Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

Reviewer: 1) Please add directions for how to opt-out of personalization on Bing.
2) For clarity, I suggest starting a new paragraph for how to view/delete imported data.
3) This link "[Microsoft Privacy Dashboard]" takes me to myaccount.microsoft.com and makes me resign into my account. Once i land on that page, it still is not obvious how to view/delete the imported data. I suggest using a more direct link to this data. I also suggest including step by step instructions to delete the data due to the ambiguity of the link.

```suggestion
* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  

To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
```

Author: separated paragraphs

Author: Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410053855) for Point 1
Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410055033) for Point 3

With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.

Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 

* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.

* In Microsoft Edge settings, turn on the toggle **Privacy, search, and services** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.

> You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).

Reviewer: ```suggestion
You can manage your browsing history that's stored on Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).
```

Author: removed secure

With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.

Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 

* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.

* In Microsoft Edge settings, turn on the toggle **Privacy, search, and services** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.

> You can manage your cloud browsing history by visiting [https://account.microsoft.com](https://account.microsoft.com).

Author: ```suggestion
You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).
```

> <!-- todo: clarify scope: locally there is one set, cloud there is another.  Dashboard has documentation about browsing history that talks about local vs. cloud data -->

Author: @amatharoo1459 - which browsing history? Locally has one browsing history; cloud has a distinct browsing history.  Check wording in Dashboard about browsing history that talks about local vs. cloud data.

Reviewer 2: ![image](https://github.com/MicrosoftDocs/edge-developer/assets/137198383/e7000fe2-1fbe-4e06-adb4-9c4396ab904e)

Author: You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).

Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time. You can import your browser data to Microsoft Edge from another browser. The data includes your favorites, browsing history, cookies, autofill data, extensions, settings, and other browsing data.

With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.

Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 

* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.

* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.

> <!-- todo: add directions for how to opt-out of personalization on Bing -->

Author: @amatharoo1459 - Add directions for how to opt out of personalization on Bing.  
per point #1 in [PR comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files/b1c6d7a11cca3887a87f9790abe864231fe1d6ca#r1410027587) from A. P.

Author: see fresh Files Changed page

To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

> <!-- todo: 

Author: @amatharoo1459 - 
* Provide a more direct link to view/delete the imported data.
* Include step-by-step instructions to delete the data.

Do those b/c the existing link goes to myaccount.microsoft.com and requires re-sign in to my account.  At that page, it still is not obvious how to view/delete the imported data.

per Point #3 in [PR Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files/b1c6d7a11cca3887a87f9790abe864231fe1d6ca#r1410027587)
